projects/ngrx.io/package.json

```diff
@@ -123,7 +123,7 @@
     "hast-util-is-element": "^1.0.0",
     "hast-util-to-string": "^1.0.0",
     "html": "^1.0.0",
-    "http-server": "^0.9.0",
+    "http-server": "^0.11.1",
```

Comment:
We had to update these because of a dependency removal - for more info see https://github.com/indexzero/http-server/issues/521